packages/client/src/App.tsx

```diff
@@ -97,116 +97,16 @@ function createRedirect(from: string, to: string) {
     }
   }
 }
-export const routesConfig = config.FEATURES.V2_EVENTS
+
+// By default we now load V2 events in the '/' route. This is a temporary flag
+// to turn off V2 events in case IET wants to switch back to V1 events.
+// This will be removed after a few releases when V2 events will be finalized.
+const turnOffV2Events =
+  typeof config.FEATURES.V2_EVENTS === 'boolean' &&
+  config.FEATURES.V2_EVENTS === false
+
+export const routesConfig = turnOffV2Events
```

Comment:
This is switching from checking the truthy value to the falsy value instead. Is there some benefit in doing so?

In PR summary we are saying that:
> Removed environment-based toggling from country config.

But we still seem to be supporting that?